prov/sockets/src/sock_ep.c

```diff
@@ -582,7 +582,7 @@ static ssize_t sock_tx_size_left(struct fid_ep *ep)
 		return -FI_EOPBADSTATE;
 
 	fastlock_acquire(&tx_ctx->wlock);
-	num_left = rbavail(&tx_ctx->rb)/SOCK_EP_TX_ENTRY_SZ;
+	num_left = ofi_rounddown_power_of_two(rbavail(&tx_ctx->rb)/SOCK_EP_TX_ENTRY_SZ);
```

Comment:
> Yes. Is that a problem? When we allocate ring buffer for tx_ctx, we roundup to the power of two for the given fi_tx_attr size. So this round down ensures the lower bound of the available size.

So if I start out and call `fi_tx_size_left`then I'll get `256` and if I post 4 sends but don't reap the completions then I'll get `128` instead of `252`?